script.js

```diff
@@ -0,0 +1,54 @@
+const header = document.querySelector('header');
+const div = document.createElement('div');
+const img = document.createElement('img');
+const ul = document.createElement('ul');
+let li = document.createElement('li');
+let a = document.createElement('a');
+
+function opened() {
+  div.style.display = 'block';
+}
+
+function closed() {
+  div.style.display = 'none';
+}
+
+div.className = 'extended';
+
+img.className = 'icon';
+img.src = './img/close.svg';
+div.appendChild(img);
+img.addEventListener('click', closed);
+
```

Comment:
- [ ] Kindly fix the broken image 👇🏻 

![777](https://user-images.githubusercontent.com/79058364/167512179-954eb28d-3821-4945-a42d-8765090413fa.png)

- [OPTIONAL ] What if you prevent the under background to scroll when the navigation bar is open you can use `over-flow: none`

Reply:
Fixes completed:
- Mob menu Display = 'none' for desktop version.
- Stop scrolling when the menu is opened
- decreased blur affect value
- 'close' icon is working through all my checks. Please have a look at your settings.
- [my partner's commit](https://github.com/petushka1/microverse-m1-w1-d3-portfolio-production/pull/11/commits/e8f3ecd4afd5fc2ce69b3779a133566519916800). We have been working on both of our projects 9+ hours.
- Edited README file:
   - listed all contributors with links 
   - Steps are described initially in the "Getting Started" section.

Reply:
Great job 💯 🥇